build/docker/run.sh

```diff
@@ -11,7 +11,7 @@ fi
 if [ ${TLS:-0} -eq 1 ]
 then
     /FalkorDB/build/docker/gen-certs.sh
-    redis-server --protected-mode no ${REDIS_ARGS:-""} \
+    redis-server --protected-mode no ${REDIS_ARGS} \
```

Comment:
there is more like this in line 20 and 23

Reply:
They weren't causing issues as they are at the end of the line, but I'll remove them nonetheless